pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[build-system]
+requires = ["setuptools", "lsst-versions >= 1.3.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lsst-multiprofit"
+description = "Astronomical image and source model fitting code."
+license = {file = "LICENSE"}
+readme = "README.rst"
+authors = [
+    {name="Rubin Observatory Data Management", email="dm-admin@lists.lsst.org"},
+]
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+keywords = ["astronomy", "astrophysics", "lsst"]
+requires-python = ">=3.10.0"
+dependencies = [
+    "astropy",
+    "gauss2d",
+    "gauss2dfit",
+    "lsst-pex-config",
+    "lsst-utils",
+    "importlib_resources",
+    "matplotlib",
+    "numpy",
+    "pydantic",
+    "pytest",
```

Comment:
Test dependencies should not be in the core dependencies. You don't need to install pytest to use this package.